Projects/Claims/Sources/SubmitClaimState.swift

```diff
@@ -18,6 +18,8 @@ public struct SubmitClaimsState: StateProtocol {
     var failedStep: FlowClaimFailedStepModel?
     var audioRecordingStep: FlowClaimAudioRecordingStepModel?
 
+    enum CodingKeys: CodingKey {}
```

Comment:
We use stores to save data. And we define what properties we want the save here and in this case we don't want to save anything :)

Reply:
Oh, this is clever. @StylianosGakis I would say this is pretty equivalent to kotlinx serializations @Transient. Considering that maybe we should have a property wrapper for this? What do you think @sladan-hedvig @julia-andersson could do something like:

```
/// A property that is Codable but never gets encoded or decoded
@propertyWrapper
public struct Transient<Value>: Codable where Value: Codable {
    public var wrappedValue: Value {
        get {
            innerValue ?? defaultValue
        }
        set {
            innerValue = newValue
        }
    }
    public var innerValue: Value?
    public var defaultValue: Value
        
    public init(wrappedValue: Value?, defaultValue: Value) {
        self.defaultValue = defaultValue
        self.innerValue = wrappedValue
    }
    
    public init(defaultValue: Value) {
        self.defaultValue = defaultValue
        self.innerValue = nil
    }
    
    enum CodingKeys: CodingKey {
        case defaultValue
    }
}


struct SomeCodable: Codable {
    @Transient(defaultValue: "test") var hello: String
    var something: String
}
```

We have a few other cases where it would be nice to just say "don't persist this".

Reply:
Oh we can do wrapper :) give me a sec

Reply:
Done 👍

Reply:
⭐

Reply:
I thought I was doing a little innocent question, but there was so much more behind this than I first thought 😅 Thanks for the explanation!